providers/openfeature-provider-flagd/pyproject.toml

```
@@ -28,13 +28,21 @@ Homepage = "https://github.com/open-feature/python-sdk-contrib"
 
 [tool.hatch]
 
+[[tool.hatch.build.hooks.build-scripts.scripts]]
+commands = [
+  "./scripts/gen_protos.sh",
```

```suggestion
  "buf generate buf.build/open-feature/flagd --template buf.gen.python.yaml",
  "sed -i.bak 's/^from schema.v1 import/from . import/' proto/schema/v1/*.py",
  "rm proto/schema/v1/*.bak",
```
then you don't need to change the `gen_protos.sh` file content